src/collection.ts

```diff
@@ -1400,7 +1400,7 @@ export class Collection<TSchema extends Document = Document> {
     if ('function' === typeof options) (callback = options), (options = {});
     // Out must always be defined (make sure we don't break weirdly on pre 1.8+ servers)
```

Comment:
This seems like something we should investigate as we only support 2.6+ servers. Commenting to remember to ticket.

Reply:
NODE-3339